### DISCUSSION TITLE
Introduce yourself

### DISCUSSION THREAD

**Discussion by amenk** [OpenPlaceGuide/openplaceguide#15] [Show and tell]
You are interested in the OpenPlaceGuide project?

Tell us about yourself and your motivation.

Which city / country would you like to apply this idea to?

**Comment by alazar-tekle** [comment]
Ethiopia

**Comment by xeruf** [comment]
Heyo, I got into OSM Mapping a while ago as I started hiking and doing bike tours, and especially now as I live almost entirely Google-Free.
I want to be able to find POIs without ads and prompts and without personal data collection.

I am from Germany > Bavaria > Bayreuth, but want to have it globally - think of going on a holiday or searching for a place to live and wanting to check local offers.
Also, I want a playful edition - https://www.openstreetbrowser.org is already pretty cool but is missing some polish, and requires too much zooming in. Interestingly I just observed that GMaps adjusts the zoom depending on your query: compare "restaurants" and "gas station". But there it does not feel clunky, it feels natural. 
I made a very simple demo using Overpass and some custom data already: http://unternehmer.land

**Reply by amenk** [reply]
Nice to meet you :-) We are currently thinking of using https://osmapp.org/ as a base for main page (and contribute missing features). As for the place pages, we are developing https://github.com/OpenPlaceGuide/opg-pages which would provide a micropage for each relevant POI in OSM and enrich it with information from (currently) a git repository, which is open for anybody to contribute pictures, restaurant menus and so on.

**Reply by amenk** [reply]
@xeruf it's online now at https://map.et

**Comment by amenk** [comment]
I am Alex, from Germany around Mainz - but I have a strong connection with Ethiopia :-) 

I started getting really into OpenStreetMap when I first set foot in Ethiopia in 2007/2008 where was no map at all on Google. We started giving GPS loggers to taxi drivers to get a coverage of the main roads for OSM and also started [AddisMap.com](addismap.com) with the aim to sustain the efforts to improve the map by business listings and make places easier to discover.

AddisMap.com is technologically a bit outdated and blends proprietary data with the OpenStreetMap maps. We want to relaunch AddisMap fully open source base (this time also making the business data fully open for contributions) and look for people who want to to the same in other geographical areas or maybe also on a thematic base.
